.gitignore

```diff
@@ -97,3 +97,4 @@ AlphaWalletTests/Snapshots/FailureDiffs
 .idea/
 /.bundle
 /vendor
+/.ruby-version
```

Comment:
Seems like `.ruby-version` should be committed, not ignored.

Reply:
This will prevent people from having different versions of Ruby. If someone updates their version, every one will have to update to this version if they are using `rbenv` to manage this directory.

Reply:
`rbenv` lets developers install multiple versions of ruby and activate a specific version in each project. `3.0.3` is the known version that works. Take a counter example, if they use ruby 2.x instead, `make` will fail. So we do want them to install and use `3.0.3`. If we update to a newer version that works and commits it, then yes, everyone should use that version. That's the point of `rbenv`; each projects pins to a specific version (ditto for `Gemfile.lock`, `Podfile.lock`).